app/assets/stylesheets/common.css

```diff
@@ -47,4 +47,10 @@ a:focus {
 .logoContainer {
   border-top: 1px solid var(--border-color);
   border-bottom: 1px solid var(--border-color);
-}
\ No newline at end of file
+}
+
+
+.su-underline {
+  --underline-color: #adb5bd;
```

Comment:
Where does this color come from? I don't see it in https://identity.stanford.edu/design-elements/color/accent-colors/

Reply:
I got it from the component library. https://github.com/sul-dlss/component-library/blob/349ef122c2724f7375e0d5cd686a650c4550d657/styles/links.css#L24

Reply:
We're changing that to stanford 40% black: https://github.com/sul-dlss/component-library/pull/119

Reply:
Sounds good. Updated.